src/renderer/screens/Balances/NetworkBalances/NetworkBalances.tsx

```diff
@@ -70,20 +71,37 @@ const NetworkBalances = ({
       return acc;
     }, {}) || {};
 
-  const filteredAssets = chain.assets.filter((asset) => {
-    if (query) {
-      return (
-        includes(asset.symbol, query) ||
-        (!searchSymbolOnly && (includes(chain.name, query) || includes(asset.name, query)))
-      );
-    }
-
-    const balance = balancesObject[asset.assetId];
-
-    return (
-      !hideZeroBalance || !balance || balance.verified !== true || (balance && totalAmount(balance) !== ZERO_BALANCE)
-    );
-  });
+  const filteredAssets = useMemo(
+    () =>
+      chain.assets
+        .filter((asset) => {
+          if (query) {
+            return (
+              includes(asset.symbol, query) ||
+              (!searchSymbolOnly && (includes(chain.name, query) || includes(asset.name, query)))
+            );
+          }
+
+          const balance = balancesObject[asset.assetId];
+
+          return (
+            !hideZeroBalance ||
+            !balance ||
+            balance.verified !== true ||
+            (balance && totalAmount(balance) !== ZERO_BALANCE)
+          );
+        })
+        .sort((a, b) => {
+          const aTotal = balancesObject[a.assetId.toString()] ? totalAmount(balancesObject[a.assetId.toString()]) : '0';
+          const bTotal = balancesObject[b.assetId.toString()] ? totalAmount(balancesObject[b.assetId.toString()]) : '0';
+
+          const aFormatted = formatBalance(aTotal, a.precision);
+          const bFormatted = formatBalance(bTotal, b.precision);
+
+          return Number(bFormatted.value) - Number(aFormatted.value) || a.name.localeCompare(b.name);
```

Comment:
`value` is a shortened value and 10 will be higher than 1M, I am not sure about requirements but it looks like total amount should be compared